src/ProcessHosts/Elastic.ProcessHosts/Elasticsearch/Process/ElasticsearchProcess.cs

```diff
@@ -82,6 +82,7 @@ protected void CheckForBadEnvironmentVariables(ElasticsearchEnvironmentConfigura
 			ErrorIfExists(c, errors, "ES_USE_IPV4", (k,v) => $"{k}={v}: set -Djava.net.preferIPv4Stack=true {i} \"-Djava.net.preferIPv4Stack=true\" {t}");
 			ErrorIfExists(c, errors, "ES_GC_OPTS", (k, v) => $"{k}={v}: set %ES_GC_OPTS: = and % {i} \"{v}\" {t}");
 			ErrorIfExists(c, errors, "ES_GC_LOG_FILE", (k,v) => $"{k}={v}: set -Xloggc:{v} {i} \"-Xloggc:{v}\" {t}");
+			ErrorIfExists(c, errors, "ES_CONF", (k,v) => $"{k}={v}: set CONF_DIR={v}");
```

Comment:
This should probably not be an `StartupException` since old installations set this environment variable.